File: testdata/jennies/builders/package-with-dashes/PHPBuilder/src/Builderpkg/SomeNiceBuilderBuilder.php
<?php

namespace Grafana\Foundation\Builderpkg;

/**
 * @implements \Grafana\Foundation\Cog\Builder<\Grafana\Foundation\Withdashes\SomeStruct>
 */
class SomeNiceBuilderBuilder implements \Grafana\Foundation\Cog\Builder
{
    protected \Grafana\Foundation\Withdashes\SomeStruct $internal;

    public function __construct()
    {
    	$this->internal = new \Grafana\Foundation\Withdashes\SomeStruct();
    }

    /**
     * Builds the object.
     * @return \Grafana\Foundation\Withdashes\SomeStruct
     */
    public function build()
    {
        return $this->internal;
    }

    public function title(string $title): static
    {
        $this->internal->title = $title;
    
        return $this;
    }

}

File: testdata/jennies/builders/package-with-dashes/JavaBuilder/builderpkg/SomeNiceBuilderBuilder.java
package builder-pkg;

import with-dashes.SomeStruct;

public class SomeNiceBuilderBuilder implements cog.Builder<SomeStruct> {
    protected final SomeStruct internal;
    
    public SomeNiceBuilderBuilder() {
        this.internal = new SomeStruct();
    }
    public SomeNiceBuilderBuilder title(String title) {
        this.internal.title = title;
        return this;
    }
    public SomeStruct build() {
        return this.internal;
    }
}
